crates/router/src/connector/cybersource/transformers.rs

```diff
@@ -93,7 +88,7 @@ impl TryFrom<&types::SetupMandateRouterData> for CybersourceZeroMandateRequest {
 
         let order_information = OrderInformationWithBill {
             amount_details: Amount {
-                total_amount: "0".to_string(),
+                total_amount: StringMajorUnit::new("0".to_string()),
```

Comment:
```suggestion
                total_amount: StringMajorUnit::zero(),
```

And can you also make `StringMajorUnit::new()` private for now?

Reply:
sure